lease/lessor_test.go

```diff
@@ -140,14 +140,14 @@ func TestLessorRecover(t *testing.T) {
 
 	// Create a new lessor with the same backend
 	nle := NewLessor(1, be, &fakeDeleteable{})
-	nl1 := nle.get(l1.id)
-	if nl1 == nil || nl1.ttl != l1.ttl {
-		t.Errorf("nl1 = %v, want nl1.TTL= %d", l1.ttl)
+	nl1 := nle.get(l1.ID)
+	if nl1 == nil || nl1.TTL != l1.TTL {
+		t.Errorf("nl1 = %v, want nl1.TTL= %d", nl1.TTL, l1.TTL)
```

Comment:
one is a pointer type, the other one is an integer. nl1 can be nil or not nil. we only care about ttl if it is not nil.